mitodl/ocw-hugo-themes
# fix: inconsistent video player styling

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/813

#### What's this PR do?
- Apply/Moves `course-v2` video changes/styling to `base-theme` so it can also be used by `course-v1` theme.

#### How should this be manually tested?
- Checkout this branch
- Build any course which has videos on `course-v1` theme
- Verify that video styling is fine and consistent.
- Build any course which has videos on `course-v2` theme
- Verify that video styling is fine and consistent. 
- Test any other use-case that needs to be tested in order to verify that these changes have no side-effect.

#### Screenshots (if appropriate)

`course v1 theme`

<img width="605" alt="image" src="https://user-images.githubusercontent.com/93309234/194837122-eb624a34-bd9d-4486-8546-2d5e11da8428.png">

`course v2 theme`

<img width="823" alt="image" src="https://user-images.githubusercontent.com/93309234/194837321-34fdaa66-9e51-4913-ac4f-d0859f7cc6be.png">
